docs/syntax/requires.md

```diff
@@ -106,13 +106,55 @@ The way to do this is using curly braces around the function name that you want
 - Note the lack of pipes (`|`). Functions are processed entirely differently than items/categories used as requirements.
 - Doing this will tell Manual that the function will either return a requires string to be processed, or will return true/false based on whether this requirement was met.
 
-Requirement functions can have no function arguments, or have any number of function arguments separated by commas. 
+Requirement functions can have no function arguments, or have any number of function arguments separated by commas.
 
 - Example with no function arguments: https://github.com/ManualForArchipelago/Manual/blob/main/src/hooks/Rules.py#L8-L15.
 - Example with one argument, add str arguments to the end of the function for more: https://github.com/ManualForArchipelago/Manual/blob/main/src/hooks/Rules.py#L17-L24
 
-Additionally, those functions can themselves return a dynamically-created requires string, which would then be processed normally in the spot where the function call was. 
+Additionally, those functions can themselves return a dynamically-created requires string, which would then be processed normally in the spot where the function call was.
 
 - Example of a returned requires string: https://github.com/ManualForArchipelago/Manual/blob/main/src/hooks/Rules.py#L26-L29
 
-\* By default, the only two generic requirement functions that we provide are the OptOne and OptAll, which are both focused on allowing a required item or items to pass the requirement even if the item(s) have been disabled through defined category options. The rest of the functions in the Rules hook file are examples.
\ No newline at end of file
+\* By default, the only two generic requirement functions that we provide are the OptOne and OptAll, which are both focused on allowing a required item or items to pass the requirement even if the item(s) have been disabled through defined category options. The rest of the functions in the Rules hook file are examples.
+
+## Bundled functions
+
+Manual comes with some helpful functions built in:
+
+### `OptOne(ItemName)`
+
+Requires an item only if that item exists.  Useful if an item might have been disabled by a yaml option.
+
+### `OptAll(ItemName)`
+
+I'm not sure how to document this function, nico pls help
+
+### `YamlEnabled(option_name)` and `YamlDisabled(option_name)`
+
+These allow you to check yaml options within your logic.
+
+You might use this to allow glitches
+
+```json
+{
+    "name": "Item on Cliff",
+    "requires": "|Double Jump| or {YamlEnabled(allow_hard_glitches)}"
+}
+```
+
+Or make key items optional
+
+```json
+{
+    "name": "Hidden Item in Pokemon",
+    "requires": "|Itemfinder| or {YamlDisabled(require_itemfinder)}"
+}
+```
+
+You can even combine the two in complex ways
+
+```json
+{
+    "name": "This is probably a region",
+    "requires": "({YamlEnabled(easy_mode)} and |Gravity|) or ({YamlDisabled(easy_mode)} and |Jump| and |Blizzard| and |Water|)"
+}

```

Comment:
Maybe also include {ItemValue(Value:count)} in this doc

Reply:
Oh, definitely.